package.json

```
@@ -9,7 +9,7 @@
     },
     "scripts": {
         "prepare": "husky && yarn build",
-        "lint": "eslint packages/ scripts/ --ext=js,mjs,ts,only,skip",
+        "lint": "eslint packages/ scripts/",
```

```suggestion
        "lint": "eslint .",
```
Might as well hit everything!